### ISSUE TITLE
describe fails with a DataFrame converted from an array

### ISSUE THREAD

**Issue by jlep** [JuliaStats/DataFrames.jl#781]
```
julia> df=convert(DataFrame,ones(2,2));
julia> describe(df)
x1
ERROR: `describe` has no method matching describe(::TTY, ::Array{Float64,1})
 in describe at /home/jao/.julia/v0.3/DataFrames/src/abstractdataframe/abstractdataframe.jl:401
 in describe at /home/jao/.julia/v0.3/DataFrames/src/abstractdataframe/abstractdataframe.jl:397
```
My julia version is 0.3.6

**Comment by johnmyleswhite** [comment]
I think conversion isn't likely to be the problem: it's probably an assumption in `describe` that every column is a `DataVector`.

**Comment by jlep** [comment]
OK, thanks for the tip. I looked at the code and it seems that this could be fixed by adding describe() methods for AbstractVectors that would simply convert the vectors to DataVectors and call the existing methods. Another solution would be to refactor the methods so that NAs are taken into account only when handling DataVectors. What would be the best way to fix this?

**Comment by jlep** [comment]
Well, my last comment didn't make much sense - I'm not too familiar with the type system in julia... Are there any problems with the fix I propose in #782?

**Comment by johnmyleswhite** [comment]
Let's switch conversation to #782. Using dispatch to handle DataVectors separately from other AbstractVectors is probably the best approach.
